datastore/gcloud/rest/datastore/datastore.py

```diff
@@ -0,0 +1,412 @@
+import logging
```

Comment:
I didn't make many changes to the code from gcloud-aio - mostly just removed async/await. In this file, I replaced aiohttp with requests.

The `unused-imports` are here because they are only used in type annotations and the linter doesn't realize that.